packages/docs/src/components/sidebar/navigationRoutes.ts

```diff
@@ -342,6 +342,10 @@ export const navigationRoutes: NavigationRoute[] = [
         name: 'dropdown',
```

Comment:
I'm not sure if that's of any importance, but non selectable non category items are slightly smaller.